README.md

```diff
@@ -2,7 +2,18 @@
 
 __`lava-dl`__ is a library of deep learning tools within Lava that support  offline training, online training and inference methods for various Deep Event-Based Networks.
 
-The library presently consists of 
+There are two main strategies for training Deep Event-Based Networks: _direct training_ and _ANN to SNN converison_. 
+
+Directly training the network utilizes the information of precise events in time. It is very accurate and results in efficient networks. However, the training these networks take a lot of time and resources.
```

Comment:
However, training these networks takes...

Reply:
Thx :)

Reply:
Directly training the network utilizes the information of precise timing of events. Direct training is very accurate and results in efficient networks. However, directly training networks take a lot of time and resources.